lib/rack/utils.rb

```diff
@@ -574,7 +574,8 @@ def names
 
     def status_code(status)
       if status.is_a?(Symbol)
-        SYMBOL_TO_STATUS_CODE[status] || 500
+        raise ArgumentError, "Unrecognized status_code symbol" unless SYMBOL_TO_STATUS_CODE[status]
```

Comment:
Wouldn't it be better to include the symbol itself?

    "Unrecognized status_code symbol #{status.inspect}"